### PR TITLE
Pacesetters UI Changes

### PR DESCRIPTION
**Pull request by globusharris**
DO NOT MERGE

### CODE REVIEW

**Comment by globusharris** [comment]
Ready to merge! I think!

**Comment by globusharris** [comment]
Note that this is good enough for pacesetters demo but table width and submission still won't work for bwwc.

**Comment by lucyq** [comment]
Ready for review @frederickjansen

File: client/app/controllers/tableController.js [hunk 1]
```diff
@@ -708,8 +708,9 @@ define(['jquery', 'Handsontable', 'table_template', 'filesaver', 'alertify', 'qt
 
   function updateTableWidth(maxWidth) {
 
-    $('#instructions').css('width', maxWidth);
-    $('#instructions').css('max-width', maxWidth);
+    //$('#instructions').css('width', '1153px');
+    $('#instructions').css('max-width', '1153px');
```
Comment (frederickjansen) [inline client/app/controllers/tableController.js:712]: Let me see if I can get rid of this hardcoded value.

File: client/app/controllers/tableController.js [hunk 2]
```diff
@@ -976,19 +977,22 @@ define(['jquery', 'Handsontable', 'table_template', 'filesaver', 'alertify', 'qt
   }
 
   function updateWidth(tables) {
+    console.log("updating");
```
Comment (frederickjansen) [inline client/app/controllers/tableController.js:980]: Please don't keep these console logs

File: client/app/controllers/tableController.js [hunk 3]
```diff
@@ -976,19 +977,22 @@ define(['jquery', 'Handsontable', 'table_template', 'filesaver', 'alertify', 'qt
...
 
-    var maxWidth = $('#instructions').width;
-
+    var maxWidth = $('#instructions').width();
     for (var i = 0; i < tables.length; i++) {
 
       var t = tables[i];
 
       var w = getWidth(t) + getHeaderWidth(t);
+      console.log(w);
```
Comment (frederickjansen) [inline client/app/controllers/tableController.js:988]: Same thing here

File: client/app/controllers/tableController.js [hunk 4]
```diff
@@ -976,19 +977,22 @@ define(['jquery', 'Handsontable', 'table_template', 'filesaver', 'alertify', 'qt
...
 
       t.updateSettings({
-        width: w,
+        width: w
       });
 
+      console.log(w);
```
Comment (frederickjansen) [inline client/app/controllers/tableController.js:994]: Same thing here

File: client/index.html [hunk 5]
```diff
@@ -117,7 +117,7 @@ <h2 class="text-center">Input your data</h2>
       <h2 class="text-center">View your data</h2>
       <p class="text-center">Your data will appear here after you drag/drop or browse to find your completed Excel
         template file above.</p>
-      <span id="expand-table-button" class="arrow-down flip"></span>
+      <span id="expand-table-button" class="arrow-down"></span>
```
Comment (frederickjansen) [inline client/index.html:120]: This will break for BWWC, since their tables are supposed to start collapsed. I guess this should become a config option?

**Review by frederickjansen — CHANGES_REQUESTED** [review]

**Comment by frederickjansen** [comment]
I'll merge for now and make changes after the training session.

**Pull request merged**